.github/workflows/publish_internal.yaml

```diff
@@ -37,8 +40,8 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           ISSUE_NUMBER: ${{ github.event.number }}
           PR_LABELS: "${{ join(github.event.pull_request.labels.*.name) }}"
-        run: dart pkgs/firehose/bin/firehose.dart --validate
+        run: dart pkgs/firehose/bin/firehose.dart --validate ${{ fromJSON('{"true":"--use-flutter","false":"--no-use-flutter"}')[env.use-flutter] }}
```

Comment:
Why is this necessary if env is always false?

Reply:
Ah nevermind, the publish internal is for testing the trick.

Weird that the && doesn't work!

Reply:
Question: didn't && and || and my PR work? The linked action seems to be using the old version (--use-flutter false) which is incorrect.